README.md

```diff
@@ -10,9 +10,11 @@
 
 WordPress plugin for adding a custom header image to posts. Only supports the block editor. There is no support for the classic editor.
 
+To enable add post type support: `add_post_type_support( 'post', 'post-header-image' );`
```

Comment:
I think post should remain as default, similar to https://github.com/Automattic/jetpack/blob/792b26b5539d07cc35fdd93567942f2ad481eef2/modules/publicize/publicize.php#L787-L794